src/styles/blocks/button.scss

```diff
@@ -0,0 +1,45 @@
+.button {
+  display: block;
+  width: 166px;
+  height: 40px;
+  margin-left: 17px;
+  margin-top: 16px;
+  border: none;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  &:hover {
+    display: block;
+    width: 166px;
+    height: 40px;
+    margin-left: 17px;
+    margin-top: 16px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    background-color: #ffffff;
```

Comment:
There is no need to repeat the non-hover state properties inside the &:hover selector. Only include the properties that change on hover. This will make the code cleaner and easier to maintain.